hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalReplicatedMapStatsImpl.java

```diff
@@ -235,62 +232,68 @@ public long getPutOperationCount() {
         return putCount;
     }
 
-    public void incrementPuts(long latency) {
+    public void incrementPutsNanos(long latencyNanos) {
         PUT_COUNT.incrementAndGet(this);
-        TOTAL_PUT_LATENCIES.addAndGet(this, latency);
-        setMax(this, MAX_PUT_LATENCY, latency);
+        TOTAL_PUT_LATENCIES.addAndGet(this, latencyNanos);
+        setMax(this, MAX_PUT_LATENCY, latencyNanos);
     }
 
     @Override
     public long getGetOperationCount() {
         return getCount;
     }
 
-    public void incrementGets(long latency) {
+    public void incrementGetsNanos(long latencyNanos) {
         GET_COUNT.incrementAndGet(this);
-        TOTAL_GET_LATENCIES.addAndGet(this, latency);
-        setMax(this, MAX_GET_LATENCY, latency);
+        TOTAL_GET_LATENCIES.addAndGet(this, latencyNanos);
+        setMax(this, MAX_GET_LATENCY, latencyNanos);
     }
 
     @Override
     public long getRemoveOperationCount() {
         return removeCount;
     }
 
-    public void incrementRemoves(long latency) {
+    public void incrementRemovesNanos(long latencyNanos) {
         REMOVE_COUNT.incrementAndGet(this);
-        TOTAL_REMOVE_LATENCIES.addAndGet(this, latency);
-        setMax(this, MAX_REMOVE_LATENCY, latency);
+        TOTAL_REMOVE_LATENCIES.addAndGet(this, latencyNanos);
+        setMax(this, MAX_REMOVE_LATENCY, latencyNanos);
     }
 
+    @Probe(name = REPLICATED_MAP_METRIC_TOTAL_PUT_LATENCIES, unit = MS)
     @Override
     public long getTotalPutLatency() {
-        return totalPutLatencies;
+        return convertNanosToMillis(totalPutLatenciesNanos);
     }
 
+    @Probe(name = REPLICATED_MAP_METRIC_TOTAL_GET_LATENCIES, unit = MS)
     @Override
     public long getTotalGetLatency() {
-        return totalGetLatencies;
+        return convertNanosToMillis(totalGetLatenciesNanos);
     }
 
+    @Probe(name = REPLICATED_MAP_METRIC_TOTAL_REMOVE_LATENCIES, unit = MS)
     @Override
     public long getTotalRemoveLatency() {
-        return totalRemoveLatencies;
+        return convertNanosToMillis(totalRemoveLatenciesNanos);
     }
 
+    @Probe(name = REPLICATED_MAP_MAX_GET_LATENCY, unit = MS)
     @Override
     public long getMaxPutLatency() {
-        return maxPutLatency;
+        return maxPutLatencyNanos;
```

Comment:
These should be converted to `MS` too.

Reply:
Nice catch. I don't know why I didn't notice this. 😅